sdk/formrecognizer/Azure.AI.FormRecognizer/src/Appearance.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Azure.AI.FormRecognizer
 {
-    /// <inheritdoc />
+    /// <summary> An object representing the appearance of the text line. </summary>
```

Comment:
If this comment is exactly the same as generated you can just remove it and that would keep the generated comment.